sympy/stats/tests/test_continuous_rv.py

```diff
@@ -1497,3 +1497,15 @@ def test_ContinuousDistributionHandmade():
     assert median(space.value) == Interval(1, 2)
     assert E(space.value) == Rational(3, 2)
     assert variance(space.value) == Rational(13, 12)
+
+def test_issue_16318():
+    #test compute_expectation function of the SingleContinuousDomain
+    x,y = symbols('x y')
```

Comment:
Keep a space after comma and before y according to PEP8

Reply:
Thank you for your feedback! I removed the section as x, y are already defined above.